src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.ts

```diff
@@ -13,26 +21,29 @@ import {VariableService} from '../../../../services/variable.service';
 })
 export class GbVariablesComponent implements OnInit {
 
-  public allChecked: boolean;
+  private _allChecked: boolean;
```

Comment:
do you need this field? it seems to me like setter and getter only will do.

Reply:
since there are extra processes inside these getter and setter, yes, this _allChecked field is necessary. To bind an Angular directive model to a component field, the field itself should be static, otherwise you may experience unstable behavior on UI.